### PR TITLE
Minor detail fixes for Developer Guide

### PR DESCRIPTION
**Pull request by jamietan2002**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -569,13 +569,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 | `*`      | user                      | tag a food stall with a tag                                                                                                  | categorize food stalls effectively                                                                                                                     | Yes                                               |
-| `*`      | user                      | list out food place according to given tag                                                                                   | get an overview of the food place with the tag I am interested in                                                                                      | Yes                                               |
+| `*`      | user                      | list out food stalls that contain a given tag                                                                                | find all food stalls that contain the tag I am Int                                                                                                     | Yes                                               |
```
Comment (Hikoya) [inline docs/DeveloperGuide.md:572]: Perhaps you left out something? I am Int does not sound right.

**Review by Hikoya — CHANGES_REQUESTED** [review]
A single nit spotted but otherwise thanks for the hard work.

**Comment by clarence-chew** [comment]
Ran a spellchecker for American English (rationale - code is done with American English so we should be consistent)

* [ ] Under DG file format "capitalisation" should be "capitalization"

* [ ] Under Use cases, impatient user, "visualization"

* [ ] Non-functional requirements > Accessibility, point 9 > "recognizable"

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jamietan2002) [reply docs/DeveloperGuide.md]: thanks for pointing out, fixed

**Review by Hikoya — APPROVED** [review]
LGTM.

**Pull request merged**